redash/query_runner/pg.py

```diff
@@ -140,7 +140,7 @@ def _get_tables(self, schema):
         AND a.attnum > 0
         AND NOT a.attisdropped
         JOIN pg_type t
-        ON c.reltype = t.oid
+        ON a.atttypid = t.oid
```

Comment:
Interesting, what was the difference here?

Reply:
It was showing the table name instead of the column type. Check it out with the Redash metadata data source in the stage or RC environment.